src/docs/issue-platform.mdx

```diff
@@ -0,0 +1,105 @@
+
+To use the Issue Platform you need to:
+1. <Link to="#writing-a-detector">Write a Detector</Link> 
+2. <Link to="#register-an-issue-type">Register an Issue Type</Link> 
+3. <Link to="#send-occurrences-to-the-issue-platform">Send Occurrences to the Issue Platform</Link> 
+4. <Link to="#Releasing-your-issue-type">Release</Link>
+
+## Writing a Detector
+The first step to introducing a new issue type to Sentry is to determine what type of issue you want to detect. It can be from any data stream, as long as you can produce an <Link to="#issue-occurrence-schema">Occurrence</Link> representing the issue
+
+## Register an Issue Type
+To register a new issue type you need to subclass `GroupType` https://github.com/getsentry/sentry/blob/ef76b0765f6c057af12b1cd0dc2de6b42a938c02/src/sentry/issues/grouptype.py#L98-L108
+
+When registering the class, set the following required attributes on the class:
+- `type_id`: A unique integer to identify your issue type.
+- `slug`: A unique slug to identify your issue type. Used to search by issue type in the issue stream
+- `description`: A human readable description of your issue type [include where this is used]
+- `category`: The overall category that your issue type belongs to. [include docs on creating a category]
+
+You may also set these optional attributes:
+- `noise_config`: A `NoiseConfig` that allows you to set policy on how many times a fingerprint must occur in a period of time before an issue is created
+
+Here's an example issue type:
+```python
+@dataclass(frozen=True)
+class ProfileFileIOGroupType(GroupType):
+    type_id = 2001
+    slug = "profile_file_io_main_thread"
+    description = "File I/O on Main Thread"
+    category = GroupCategory.PROFILE.value
+```
+
+## Send Occurrences to the Issue Platform
+If operating within the Sentry codebase, you can call <Link to="https://github.com/getsentry/sentry/blob/master/src/sentry/issues/producer.py#L28-L30">sentry.issues.produce_occurrence_to_kafka</Link> and pass in the <Link to="https://github.com/getsentry/sentry/blob/d262bbd9d8d3a2747d5251bee35c016c13ca050a/src/sentry/issues/issue_occurrence.py#L58">IssueOccurrence</Link>, and optionally the `Event` data. The fields in these are described in the schemas below.
+
+If operating from an external service then pass a json payload that matches the <Link to="#issue-occurrence-schema">Issue Occurrence Schema</Link> to the `ingest-occurrences` topic on the <Link to="https://github.com/getsentry/getsentry/blob/97039335a29517a7e48149148cdd7a30a60971e7/getsentry/conf/settings/prod.py#L567-L571">issue occurrences Kafka cluster</Link>
+
+### Issue Occurrence Schema:
+Required:
+- `id`: str. A user generated uuid in string format representing a unique id for this occurrence.
+Generally it's a good idea to generate this separately to your event id.
+- `project_id`: int. Id of the project this occurred in
+- `fingerprint`: Sequence[str].  Unique identifier for the root cause of this problem.
```

Comment:
I think you should expand into a different section how to make a fingerprint and give examples.

Reply:
Yeah agreed, I think we will have a whole separate doc about writing detectors